src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -73,6 +79,24 @@ public function getRelationFields()
         return $relationFields;
     }
 
+    /**
+     * Associate and dissociate.
+     *
+     * @param  Model
+     * @param  array The form data.
+     * @return Model Model with relationships set up.
+     */
+    public function associateBelongsToRelations($item, array $data)
+    {
+        $belongsToFields = $this->getFieldsWithRelationType('BelongsTo');
+
+        foreach ($belongsToFields as $relationField) {
+            $item->{$this->getOnlyRelationEntity($relationField)}()->associate($relationField['model']::find(Arr::get($data, $relationField['name'])));
```

Comment:
This will conflict with #3322 .

This code needs to check for association/dissociation currently only checks for association. It's fixed in #3322

Reply:
Passing null is the same as dissociation, I guess.

Reply:
Indeed, I'v just confirmed and it dissociates with null. Was not working in my tests previously, I need to know why. I will revert in the other PR if something was causing this issue other than my own error.

Thanks @mgralikowski

Reply:
Anyway, maybe I was too focused on "packing" code into single line and it lost some readability I think :)